### PR TITLE
Updated instructions based on feedback portal

### PR DESCRIPTION
**Pull request by ogardy**
Updated the instruction to reflect VS 2019 approach.
Feedback comment from 10/07/2020 14:47 Complete

### CODE REVIEW

**Review by ivo-stoilov — APPROVED** [review]
:shipit:

File: getting-started/using-telerik-justmock-in-your-test-project.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ This topic demonstrates how to use __Telerik® JustMock__ in a new test project
 	**Figure 1: Add new project**  
 	![Add New Project to VS Solution](images/AddNewProject.png)
 
-1. Select the Test Project template. Enter a Name and click OK.
+1. Select the Test Project template or search for it by typing "JustMock test". After that select the Test Project template and click OK.
```
Comment (yordan-mitev) [inline getting-started/using-telerik-justmock-in-your-test-project.md:26]:
```suggestion
1. Select the desired **JustMock Test Project** template from the menu and click OK. 
```
The first sentence instructs the reader to select the Test Project template. The second sentence duplicates the same instructions and asks the reader to  select the Test Project template again.
We can omit the template search instruction, I am not sure if the add more value.

File: getting-started/using-telerik-justmock-in-your-test-project.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ This topic demonstrates how to use __Telerik® JustMock__ in a new test project
 	**Figure 1: Add new project**  
 	![Add New Project to VS Solution](images/AddNewProject.png)
 
-1. Select the Test Project template. Enter a Name and click OK.
+1. Select the Test Project template or search for it by typing "JustMock test". After that select the Test Project template and click OK.
 
 	**Figure 2: Choose the JustMock Test Project template**  
-	![JustMock Test Project Template](images/ProjectTemplate.png)
+	![JustMock Test Project Template](images/ProjectTemplate1.png)
```
Comment (mihail-vladov) [inline getting-started/using-telerik-justmock-in-your-test-project.md:29]: Please replace the image or remove the old and add a new one with a different name if required. And please try to name the images according to their usage.

**Review by mihail-vladov — CHANGES_REQUESTED** [review]
@ogardy Could you address the comment? Thanks.

**Review by mihail-vladov — APPROVED** [review]

**Pull request merged**